src/features/unified-share-modal/__tests__/ContactsField-test.js

```diff
@@ -107,6 +149,21 @@ describe('features/unified-share-modal/ContactsField', () => {
 
             expect(options.length).toEqual(0);
         });
+
+        test('should only call addSuggestedContacts() when has suggestedCollaborators', () => {
+            const wrapper = getWrapper({
+                selectedContacts: [],
+                suggestedCollaborators: { '12345': { id: '12345' }, '23456': { id: '23456' } },
+            });
+            const addSuggestedContactsMock = jest.fn();
+
+            wrapper.setState({ pillSelectorInputValue: 'x@' });
+
+            wrapper.instance().addSuggestedContacts = addSuggestedContactsMock;
+            wrapper.instance().filterContacts(contactsFromServer);
+
+            expect(addSuggestedContactsMock).toHaveBeenCalledWith([expectedContacts[0]]);
```

Comment:
Is there a test for verifying that addSuggestedContactsMock was not called when suggestedCollaborators is empty?

Reply:
I'll add a test for that case.